packages/model/src/reminder.ts

```diff
@@ -2,11 +2,15 @@ import { ListResponse } from './common';
 import { EventResponse } from './event';
 import { ResearchOutputDataObject } from './research-output';
 
-type ReminderEntity = 'Research Output' | 'Event';
+type ReminderEntity = 'Research Output' | 'Event' | 'Event Material';
 
 type ResearchOutputReminderType = 'Published';
 type EventReminderType = 'Happening Today' | 'Happening Now';
-type ReminderType = ResearchOutputReminderType | EventReminderType;
+type EventMaterialReminderType = 'Video';
```

Comment:
with the above comment in mind I think this should move to `EventReminderType` and be a new element of the union called something like `Video Updated`